packages/plugins/persisted-operations/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@graphql-yoga/plugin-persisted-operations",
-  "version": "1.7.0",
+  "version": "1.7.1",
```

```suggestion
  "version": "1.7.0",
```
This will be updated by changesets once we release. So you shouldn't update it manually like that.